### ISSUE TITLE
Feature: Search for Devicename instead of serialnumber on physical device

### ISSUE THREAD

**Issue by onerrorgoto** [MSEndpointMgr/CloudLAPS#23]
Why can I not search for a physical computers devicename?
Why do I need to use the serialnumber?
The user seldom knows the serialnr but the computername (Devicename) is often known thru sticker.

**Comment by RaghuKeenLearner** [comment]
You can always get the serial number from MEM under the device properties but yeah name lookup would be good. But hey this is a free product and I appreciate what all this community has done for us. Good luck.

**Comment by AndersElonGroup** [comment]
I agree that the community has done an amzing job with CloudLAPS and this is just a wish for a feature update.

As a MEM admin, yes I can find the Serialnumber. But as a Helpdesk-worker with no access to MEM it is more difficult, they only have access to the CloudLAPS-portal to assist the users.

I have solved it by adding BGInfo to the desktop of all Autopilot-devices and there show the serialnr and other information.
(https://oliverkieselbach.com/2018/12/15/deploying-win32-app-bginfo-with-intune/)
